### ISSUE TITLE
🔗 Add input validation for the URL shortener form

### ISSUE THREAD

**Issue by LyubomirT** [LyubomirT/url-slicer#2]
Implement client-side validation for the URL input field. Make sure the URL you entered is valid before submitting the form.

**Comment by khanSoliheen** [comment]
@LyubomirT , We have add a regex from the frontend side to do the validation for url format. Am I right? Please assign this issue to me.

**Comment by LyubomirT** [comment]
> @LyubomirT, we have added a regex on the frontend side to validate the URL format. Is that correct? Please assign this issue to me. There are two text boxes: one for the original URL and another for the custom alias. We need to add the validation to both. While compiling the project getting this error Transporter verification error: Error: Invalid login: 535 5.7.3 Authentication unsuccessful [PN2PR01CA0033.INDPRD01.PROD.OUTLOOK.COM 2024-08-17T19:47:13.236Z 0. I have added the required values in .env.

Thank you for your interest in contributing to the project! The URL verification concept is correct, that's right. Although the issue with the email is probably a problem on your end, since everything works fine for me. Are you sure you use the required credentials?

**Comment by khanSoliheen** [comment]
My .env file look like this.
login= here should be an outlook email
password= password 
PORT=3001
SESSION_SECRET=your_session_secret // this I haven't changed.
MongoURI=mongodb://localhost:27017
Thanks for the help.
